R/stations.R

```diff
@@ -468,7 +468,7 @@ stations_search <- function(name = NULL,
 
 
 normals_stn_list <- function(yr) {
-  httr::GET(getOption("weathercan.urls.stations.normals"),
+  get_check(getOption("weathercan.urls.stations.normals"),
             query = list(yr = yr)) %>%
```

Comment:
As above, great idea!
```suggestion
  get_check(getOption("weathercan.urls.stations.normals"),
            query = list(yr = yr), task = "access normals station list") %>%
```